scripts/irods/test/test_dynamic_peps.py

```diff
@@ -502,20 +502,22 @@ def test_if_StructFileExtAndRegInp_is_exposed__issue_7413(self):
             # session collection.
             core.add_rule(dedent('''
             pep_api_struct_file_ext_and_reg_post(*INSTANCE, *COMM, *STRUCTINP) {{
-                *v = *STRUCTINP.objPath;
+                *v = *STRUCTINP.obj_path;
                 msiModAVUMetadata('-C', '{self.user.session_collection}', 'add', '{avu_prefix}_0', *v, '');
 
                 *v = *STRUCTINP.collection_path;
                 msiModAVUMetadata('-C', '{self.user.session_collection}', 'add', '{avu_prefix}_1', *v, '');
 
-                *v = *STRUCTINP.oprType;
+                *v = *STRUCTINP.opr_type;
                 msiModAVUMetadata('-C', '{self.user.session_collection}', 'add', '{avu_prefix}_2', *v, '');
 
                 *v = *STRUCTINP.flags;
                 msiModAVUMetadata('-C', '{self.user.session_collection}', 'add', '{avu_prefix}_3', *v, '');
 
                 *v = *STRUCTINP.defRescName;
                 msiModAVUMetadata('-C', '{self.user.session_collection}', 'add', '{avu_prefix}_4', *v, '');
+
+                *v = *STRUCTINP.forceFlag;
```

Comment:
Needs an `msiModAVUMetadata` call after this, I think

Reply:
In my testing, `forceFlag` didn't actually get set to a value; it was just there or not there. I thought the additional AVU would be unnecessary because we're just checking if `forceFlag` exists and not its value-- if it didn't, then this rule would error out with `UNMATCHED_KEY_OR_INDEX` and fail the assertion later.

Reply:
Oh, I see. The value is empty (because we use the force keyword as a flag rather than a key-value pair) so it can't annotate the empty value via `msiModAVUMetadata`. Is that right? Let's leave a comment saying something along those lines for future readers.

Reply:
I believe you can test for the existence of `forceFlag`. If it's there, set a custom AVU. Then the rest of the test checks to see if the custom AVU is there.

Reply:
> I believe you can test for the existence of `forceFlag`. If it's there, set a custom AVU. Then the rest of the test checks to see if the custom AVU is there.

What would the NREP syntax for this be?
```
if(*STRUCTINP.forceFlag) {
...
}
```
or such?

Reply:
Seems reasonable to me

Reply:
You can check for an empty string.
```
if ('' == *STRUCTINP.forceFlag) {
```